CIScripts/Test/Tests/DockerDriver.Tests.ps1

```diff
@@ -0,0 +1,69 @@
+Param (
+    [Parameter(Mandatory=$true)] [string] $TestbedAddr,
+    [Parameter(Mandatory=$true)] [string] $ConfigFile
+)
+
+. $PSScriptRoot\..\..\Common\Aliases.ps1
+
+. $ConfigFile
+$Session = New-PSSession -ComputerName $TestbedAddr -Credential (Get-TestbedCredential)
+$TestsPath = "C:\Artifacts\"
+
+function Start-DockerDriverUnitTest {
+    Param (
+        [Parameter(Mandatory=$true)] [PSSessionT] $Session,
+        [Parameter(Mandatory=$true)] [string] $Component
+    )
+
+    $TestFilePath = ".\" + $Component + ".test.exe"
+    $Command = @($TestFilePath, "--ginkgo.noisyPendings", "--ginkgo.failFast", "--ginkgo.progress", "--ginkgo.v", "--ginkgo.trace")
+    $Command = $Command -join " "
+
+    $Res = Invoke-Command -Session $Session -ScriptBlock {
+        Push-Location $Using:TestsPath
+
+        # Invoke-Command used as a workaround for temporary ErrorActionPreference modification
+        $Res = Invoke-Command -ScriptBlock {
+            $ErrorActionPreference = "SilentlyContinue"

```

Comment:
Refinement: Please move this $ErrorActionPreference = "SilentlyContinue" inside if.